### DISCUSSION TITLE
Dúvida com display: inline

### DISCUSSION THREAD

**Discussion by andersonmlima** [frontendbr/forum#1677] [Perguntas e Respostas]
Olá pessoal boa tarde!

Sou iniciante no mundo de programação, estou aprendendo agora mas ja me deparei com uma duvida que para vocês acredito ser bem simples rs..

https://codepen.io/anderliima/pen/xxbNKrK

![DISPLAY](https://user-images.githubusercontent.com/54910497/73289337-4df88f80-41db-11ea-8a85-9f833d8d4a41.png)

**Comment by valkervieira** [comment]
Oi, @AnderLiima, pode descrever melhor qual é a dúvida?

**Comment by andersonmlima** [comment]
@borghettinho  tudo tranquilo?

Então, como falei sou novo ainda na programação e aqui no Github, então posso me expressar mal quanto a duvida.

Esse código é o 'errado' que não aceita o display-inline na lista do header;
https://codepen.io/anderliima/pen/GRgaRKQ

Esse código, praticamente o mesmo, esta funcionando o display-inline na lista do header;
https://codepen.io/anderliima/pen/xxbNKrK

**Comment by carlosviniciusananias** [comment]
@AnderLiima seu problema esta nas propriedades que definiu na **ul**...

**Seu erro esta aqui:** 
ul {
	display: inline-block;
	vertical-align: top;
	width: 20%;
	margin-right: 15%
}

**Comment by andersonmlima** [comment]
@carlosviniciusananias  valeeeeu mesmo hein, ajudou muito hehehe corrigiu até um outro probleminha, e realmente faz sentido não ter essa ul...

Valeu mano, se você puder me indicar cursos, sites e etc para quem esta iniciando nessa carreira ficarei mais grato ainda!
